demo/app/lib/views/wallet_initiated_connect.dart

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import 'dart:convert';
+import 'dart:developer';
+import 'package:app/models/connect_issuer_config.dart';
+import 'package:app/views/supported_credentials_list.dart';
+import 'package:app/widgets/primary_button.dart';
+import 'package:flutter/material.dart';
+import 'package:app/widgets/common_title_appbar.dart';
+import 'package:flutter/services.dart';
+import 'package:app/wallet_sdk/wallet_sdk_mobile.dart';
+
+class IssuerList extends StatefulWidget {
+  const IssuerList({Key? key}) : super(key: key);
+
+  @override
+  State<IssuerList> createState() => IssuerListState();
+}
+
+class IssuerListState extends State<IssuerList> {
+  Map<String, String> issuerMetadata = {};
+  Map<String, dynamic> values = {};
+  var WalletSDKPlugin = WalletSDK();
+
+  readIssuerAuthFlowConfig() async {
+    final String configResponse = await rootBundle.loadString('lib/assets/walletConnectIssuerConfig.json');
```

Comment:
Move config loading to the service. It can be ConfigService. 
It can have a function readIssuerAuthFlowConfig that returns ConnectIssuerConfig. 
The signature of this function will look like this `Future<ConnectIssuerConfig> readIssuerAuthFlowConfig() async`

Reply:
BTW: `String redirectURI` can be added to ConnectIssuerConfig, just do not read it from JSON, read it from fromEnvironment, as you do in the code below.

Reply:
It was there in the config before but rolson suggested its same for every issuer so we should be able to read it from the enviornment so i made change accordingly

Reply:
Yes, it will be read from env variable, but it can be in ConnectIssuerConfig class

Reply:
Like this

```
return ConnectIssuerConfig(
      issuerURI: json['issuerURI'],
      scopes: json['scopes'],
      clientID: json['clientID'],
      showIssuer: json['showIssuer'],
      description: json['description'],
       redirectURI:String.fromEnvironment("redirectURI")
    );
```

Reply:
Yes got it will change